### PR TITLE
Update App.vue

### PR DESCRIPTION
**Pull request by manjaroblack**
Changed some formatting and wording

### CODE REVIEW

File: js/App.vue [hunk 1]
```diff
@@ -235,11 +230,11 @@
                 <h2 class="heading">
                     List of all {{ displayableGambits.length }} gambits sorted by
                     <select v-model="order" class="heading bg-transparent text-left">
-                        <option value="popularity">Their popularity</option>
-                        <option value="rate">Their winning rate</option>
-                        <option value="trophies">The number of trophies</option>
-                        <option value="videos">The number of videos</option>
-                        <option value="name">Their name</option>
+                        <option value="popularity">POPULARITY</option>
```
Comment (elaqqad) [inline js/App.vue]: We should not use caps here.

File: js/App.vue [hunk 2]
```diff
@@ -53,47 +53,44 @@
...
-                                    >
-                                    or
-                                    <span
-                                        class="dotted-underline text-yellow-900 cursor-pointer"
-                                        @click.prevent="formFill('lichess', 'DrNykterstein')"
-                                        >Magnus Carlsen</span
-                                    >
+                                <div class="text-sm">Examples
+                                    <ul class="list-none list-inside pl-4">
+                                        <li>Chess.com:
```
Comment (elaqqad) [inline js/App.vue:58]: There is actually a good reason I don't like to have lists here,  in terms of design and accessibility for the users, it's better that the initial form does not appear very long (in terms of height). Users should not need to scroll down in order to click "analyze my games". 

I tried to do a more suitable way in my version, even tho I still prefer the initial version: It's only one line, it's not mandatory, it does not add more space in the first form.

File: js/App.vue [hunk 3]
```diff
@@ -8,9 +8,9 @@
     >
         <div class="text-center">
             <h1 class="text-6xl md:text-8xl mb-4">
-                <a href="/"> Holy Gambit Grail ! </a>
+                <a href="/">Holy Gambit Grail!</a>
             </h1>
-            <p class="md:text-2xl">How many of these gambits have you won with ?</p>
+            <p class="md:text-2xl">How close are you to completing the Holy Gambit Grail?</p>
```
Comment (elaqqad) [inline js/App.vue]: I don't want to repeat the word Holy Gambit Grail twice. The title already indicates what we are talking about. The second phrase defines what the holy gambit grail is about for a large audience.

**Review by elaqqad — COMMENTED** [review]
I did adapt the changes that I agree with in the next version (not all of them thought).
You can still insist if you think my remarks are out of place.

File: js/App.vue (hunk 3 above)
Comment (manjaroblack) [reply js/App.vue]: Ok. I agree. How about "How many gambits have you won with?"

File: js/App.vue (hunk 1 above)
Comment (manjaroblack) [reply js/App.vue]: I think it is capitalized by the styling? I added the caps just to remind myself. Is it bad?

File: js/App.vue (hunk 2 above)
Comment (manjaroblack) [reply js/App.vue:58]: You make a good point. I just finished learning JavaScript in university a month ago, so I'm unfamiliar with best practices. Technically these contributions are my first go at it!

File: js/App.vue (hunk 3 above)
Comment (elaqqad) [reply js/App.vue]: Ok with that one.

File: js/App.vue (hunk 2 above)
Comment (elaqqad) [reply js/App.vue:58]: Thanks for the contributions! Trying and erring is the most effective way of learning.

File: js/App.vue (hunk 1 above)
Comment (elaqqad) [reply js/App.vue]: Yes, they are capitalized by the style, so if we change the style the whole phrase will not be capitalized except the words you capitalized explicitly. For coherance we need to let the style do its job (capitalize or not).

Comment (manjaroblack) [reply js/App.vue]: That makes so much sense!

**Comment by elaqqad** [comment]
For now, I am closing this PR. Because the changes were already made in the main branch.
You can fetch/rebase and create another PR if you want to suggest new changes.
Consider running "npm run format" before committing.

**Pull request closed without merging**